app/models/application_document.rb

```diff
@@ -11,6 +11,18 @@ def keywords
     metadata['Keywords']
   end
 
+  def audience
```

Comment:
I don't love this, but have come around to it because it makes it explicit that - at present - there is no metadata source for audience, content_type, or searchgov_customs for application documents.  I suspect this will not always be the case, so this also will make future updates to this logic very straightforward following the initial v1 MVP.